raster/r.univar/r.univar.html

```diff
@@ -53,7 +53,7 @@ <h3>PERFORMANCE</h3>
 <p>
 <em>r.univar</em> supports parallel processing using OpenMP. The user
 can specify the number of threads to be used with the <b>nprocs</b> parameter.
-However, parallelization is disabled when the MASK is set.
+However, parallelization is disabled when the raster mask is set.
```

Comment:
active?

Reply:
leaving "is set", but we can indeed revise how we are using it (active, set, present, ...).